### ISSUE TITLE
TAG Appointment Process Shortcomings

### ISSUE THREAD

**Issue by martinthomson** [w3c/w3process#809]
(I've already shared feedback along these lines with the Team.)

The obvious and stated purpose of TAG appointments is to address the innate drawbacks of an election process as far as diversity goes.  It does not matter what aspect of diversity is considered, elections cannot produce multiple selections in a way that ensures that the set of selections maintains an appropriate amount of diversity.  For instance, this year we have a troubling lack of women (though this is not the fault of the electors).

Having the Team be responsible for appointments on the same timeline as elections is a very poor design for several reasons.

There is insufficient time between the time that the election terminates and when the Team needs to make a decision.  From my experience with the IETF NomCom, finding a substantive enough selection of people who are willing to serve in a position like this takes time, then it takes time to gather the information necessary to make a good decision.  In this case, it will also take time to make a decision, as decisions need to be ratified by a supermajority of both the TAG and the AB[^1].

[^1]: A side note: I do not think that the AB has a role in making this particular decision.  A more typical process is for the body itself to manage diversity appointments.  If the goal is to ensure that the Team remains accountable to the membership for their choices, then put potential appointments up for comment and maybe objections (formal or otherwise) from the membership.  I will open a separate issue to track this.

Six weeks is very much inadequate for this purpose, especially considering that this is over a holiday period in many parts of the world.  I'm personally leaving on vacation soon and will not return before the end of the nomination period.  I am sure that you will find it extraordinarily difficult to find good people.

As far as I can see, there is no formal reason for the elections to run so late in the term.  You might say that if more time is needed, then the Team could schedule the election earlier so that there is more time for appointments.

I would argue instead that having elections conclude close to the time that a term starts is a good goal.  Extending the period between election and term start makes it more likely that the election results become irrelevant, as those elected are more likely to have job changes or other life events that invalidate the results.  The gap between election and term start should only allow enough time to manage the necessary logistical functions and for those elected to transition work as needed.  Six weeks is about right for this purpose, four would be better, but that timing doesn't work.  Note also that this settling period would not be available to appointees no matter how long that period is for elected members.

The other reason that this process is not ideal is that it takes very little account of what the TAG - the group to which the appointment is being made - needs.  Though the new TAG votes on any decision, I would argue that consultation with the TAG is paramount before the appointment process starts.  For new TAG members like myself, I have very little information about my soon-to-be colleagues and their strengths.  I am not confident of being able to make an informed decision if an appointment is put to the vote.

My suggestion therefore is this:

1. Start the appointment process a short period after the new TAG is seated.
2. Consult the TAG about what you think their needs are and be guided by what they have to say about the gaps that need to be addressed.
3. Run the appointment process on a schedule that ensures that the Team are able to find good candidates.
4. Have the terms for appointments staggered relative to elected appointments.  I would say that a reasonable timeline for the entire process would be at least three months, so that would have appointed members join the TAG around May or June.

**Comment by LeaVerou** [comment]
Thank you for opening this thread Martin, and huge +1 to most of this.

Another issue with the current process is that I think it was handled very poorly that *both* chairs were up for election in the same election cycle, risking leaving the TAG with no experienced chair, when there is the obvious solution of staggering this so that only one chair is up for election at any given term.

**Comment by martinthomson** [comment]
With the caveat that I haven't served on the TAG, I don't perceive the chairing thing as that big of a problem.  The TAG has multiple chairs and should have adequate experience from those who are not up for election in the current round, plus those who were appointed in the previous round.  At least to cover a shortfall of a few months, even if it means having reduced chairing capacity in the event of the election not producing suitable chairs.  While it is true that some amount of forethought in terms of making appointments would help, we have two experienced chairs who won elections this time around, so maybe chairing is not the area in which the TAG has a gap.

Nor do I regard having appointed members viewed as being outsiders as particularly serious.  Sure, that's how people are sometimes, but we should expect better from people operating at this level.

**Comment by cwilso** [comment]
What I was describing was less about  being _viewed_ as an outsider and more "not able to participate in the getting-set-up period that the rest of the group goes through".  Perhaps this is more relevant on the AB - there are three tasks the AB traditionally does on an annual cycle, shortly after a new round of members is seated:

- goes through a prioritization exercise to determine their Priorities (https://www.w3.org/wiki/AB/Priorities) for the year
- (prior to the pandemic) chose when and where the next year of FTF meetings would be.  
- the Chair(s) are chosen right after the new AB is seated

I missed out on the first two of those in the "gap year", as I didn't think I'd be on the AB.  It would be unfortunate if appointed members never were not part of those kinds of annual tasks.  Perhaps this is not as relevant for the TAG (except for the chairing) - depending on how FTF dates/locations are chosen, as well.

**Comment by LeaVerou** [comment]
> Also, to Lea's comment - if the terms were staggered in such a way, it would make appointing someone for the lack of chairing capacity in the elected members not an option, as the TAG would be chairless for 3 months.

Either my point didn't come across well or I'm not understanding your response. What I meant was, the terms should be staggered in such a away that for every election there is always one chair who keeps their seat regardless of election results or appointments. Then the TAG would not be chairless for any amount of time.

**Comment by cwilso** [comment]
Unless you restricted the choice of who could be chair, how could you prevent that?

**Comment by martinthomson** [comment]
Given that the TAG tends to have multiple chairs, I think that it would be acceptable to go with fewer than a complete set of chairs for a period.  Once in steady state, the only potential problem is that all chairs are elected and do not continue (either because they lose a vote or because they choose not to stand).  What happens then is that the TAG needs to find a chair, with - based on previous election timing - relatively short notice.  The consequence of the change I suggest is that appointments won't be able to help immediately, leaving a short period where someone else needs to step up.

I don't see any need to restrict chair choice to solve that problem.  Though there might be some (or indeed many) serving on the TAG who might prefer not to take on chair responsibilities, that would be a rare failure case.  If we do rely on appointments to cover a gap in chair-willing members, then every appointment cycle will need to include one appointment who is at least temporarily willing to chair.  I'm not sure that the Team would be happy with that constraint, but I don't know if an elected body can avoid that class of problem, particularly when chairing is not something you can go without.

**Comment by LeaVerou** [comment]
> Unless you restricted the choice of who could be chair, how could you prevent that?

There are two chairs. If one is up for re-election or re-appointment and the other is not, I don't see what there is to prevent?

**Comment by cwilso** [comment]
Perhaps I misunderstood, @LeaVerou .  Chairs can be anyone on the TAG; the Team+TAG can select any member, or members, to be chairs each year. If they select two TAG members whose terms are the expiring after that year, I'm not sure what you're suggesting in "staggering the terms"?  They're not elected or appointed to their seats (and thus their terms, with their terms end dates) as chairs?

**Comment by dwsinger** [comment]
Right, to achieve guaranteed continuity you'd need rules that there must be co-chairs, and one must be appointed from the class who have two remaining years. Whether that restriction would be acceptable I do not know.

**Comment by LeaVerou** [comment]
Ah, I see what you're saying now. Indeed, it's still possible to end up chairless, but if we *start* by staggering the terms, it's less *likely*. Term expiration could even be taken into account in the chair selection process, like @dwsinger suggested.

**Comment by cwilso** [comment]
Well, then, I suppose it's good the current TAG chairs are now on staggered terms (if the TAG selects the same chairs, etc.)

**Comment by frivoal** [comment]
> I suppose it's good the current TAG chairs are now on staggered terms

I'm not sure that's true though. One of the two got [elected for a short term](https://lists.w3.org/Archives/Member/w3c-ac-members/2023OctDec/0059.html), and the other [had an affiliation change post election](https://w3c.social/@torgo@mastodon.social/111686116573780992), which according to [Process 3.3.3.6.](https://www.w3.org/2023/Process-20231103/#AB-TAG-vacated) means the seat gets reopen for election at the next scheduled election. So unless I missed something, they are not on staggered terms.

**Comment by cwilso** [comment]
Sort of, yeah.  I read through the Process twice, and AFAICT it could be argued that Dan is not yet a "participant" on the TAG on his new term - his affiliation change simply results in Process 3.3.3.2 bullet 1 getting triggered: "At the completion of the next regularly scheduled election for the TAG, the organization must have returned to having at most one seat."  This could also be accomplished by the other participant with the same affiliation resigning.

As this is not the first instance of affiliation changes post-election, prior-to-term-beginning in recent history, perhaps the Process needs to explicitly address this happening (in this case, it could be clarified by saying "participant _or participant-elect_" in 3.3.3.6, and 3.3.3.2 being clarified - probably by saying "of an existing participant _or participant-elect_")

Note the other case was on the Board, which has no affiliation-change semantics, which I consider a serious error.  (cf https://github.com/w3c/board/issues/18).

**Comment by chrisn** [comment]
I think we should uphold the principle of organisations not having multiple seats, and would prefer to extend this to include both appointed and elected seats. (I note that in the current situation the Team could use one of the appointed seats to allow for the affiliation change.) It's also not clear to me why the TAG and AB have significantly different rules in the Process, as for the AB there's a 30 day period, so in my view we should align on similar for the TAG.

**Comment by LeaVerou** [comment]
Off-topic, but wanted to capture these thoughts somewhere:

> [affiliation changes]

The reason this rule exists is to prevent bait-and-switch where someone runs with a certain affiliation, gets votes with that affiliation, then changes to an affiliation that may not have had the same approval rate, right?

If that assumption is correct, we could make things easier for everyone by instead introducing an approval poll. It could even include a short statement from the member. If there is lack of opposition to keep them despite the affiliation change (which I suspect will be the vast majority of the time) we leave their term as is and let them be. If borderline, we follow the current process, and if approval rate is particularly low, we do a special election.

> I think we should uphold the principle of organisations not having multiple seats, and would prefer to extend this to include both appointed and elected seats.

I would actually love to explore ways to relax that while preserving the original intent of the rule. There are very few people in the world that have both the depth and breadth of knowledge to be on the TAG and the willingness to spend the time needed, and we've lost many a great folks because of the inflexibility of this.

**Comment by chrisn** [comment]
> The reason this rule exists is to prevent bait-and-switch where someone runs with a certain affiliation, gets votes with that affiliation, then changes to an affiliation that may not have had the same approval rate, right?

Partially, but also to prevent over-representation and hence influence from people from a single company.

**Comment by LeaVerou** [comment]
> > The reason this rule exists is to prevent bait-and-switch where someone runs with a certain affiliation, gets votes with that affiliation, then changes to an affiliation that may not have had the same approval rate, right?
> 
> Partially, but also to prevent over-representation and hence influence from people from a single company.

That seems to be an orthogonal issue, but I'll add a note that I'm only talking about cases where the affiliation change wouldn't violate that other rule, for clarity.

**Comment by frivoal** [comment]
@LeaVerou

> The reason this rule exists is to prevent bait-and-switch where someone runs with a certain affiliation, gets votes with that affiliation, then changes to an affiliation that may not have had the same approval rate, right?

Yes. That's how I read it too, though "bait-and-switch" implies that this is deliberate timing by the candidate to trick voters, but the same thing can largely happen even if it is purely fortuitous. To the extent that affiliations matter at all, when they change post election then we have a question.

> If there is lack of opposition to keep them despite the affiliation change (which I suspect will be the vast majority of the time) we leave their term as is and let them be.

I think about it a bit differently depending on the situation. It maybe the that the old affiliation was deemed acceptable, and we're now wondering if the new one still is. Or it maybe be that the old affiliation was deemed desirable, and we're not wondering if the new one is equally.

If affiliation was only possibly a downside, then I guess your suggestion would make sense: after someone changes, check that they remain acceptable, and if so, keep them. But affiliation can also be a positive. If someone was voted in over other people in part _because_ of their affiliation, they don't necessarily become unacceptable when that changes, but they might be less desirable than previously, and electors might prefer other candidates over them now. Or maybe not. A competitive election let's you tell that apart, in a way an approval vote doesn't.

The obvious downsides is that elections are stressful and election campaigns are time consuming, and we want our elected representative to spend their time doing what they were elected to do, rather than spend their time trying to keep their seat.

**Comment by martinthomson** [comment]
This is getting a little off-topic.  How we deal with affiliation change is definitely worthy of more consideration, but I'd prefer if that discussion were continued on another issue.

**Comment by mnot** [comment]
Until that issue is established: :) 

I tend to agree with @LeaVerou - we over-rotate on affiliation limits. TimBL asked me to serve on the TAG in the early 2000's and I couldn't because my employer already had someone on the TAG (David Orchard). David and I often had clashing views, and I think it's safe to say that there was zero chance that our employer would have been advantaged by that. 

I agree there should be a limit - the TAG shouldn't be dominated by one company, or for that matter, one part of the industry. But the current limits are much too strict.

**Comment by cwilso** [comment]
I'm sure you've noticed that the limits are less strict than in the early 2000s - a change of affiliation does not cause an immediate event on the TAG - but you still can't add (through appointment or election) someone who shares affiliation with someone already on the TAG.  I think you're suggesting that be relaxed.  What do you think would be more appropriate, and should we have symmetry on the AB and BoD then?

**Comment by chrisn** [comment]
> I tend to agree with @LeaVerou 

If the TAG's only role were design reviews, I would also tend to agree. But with a wider governance scope, limits are appropriate.

But, there are other potential approaches that could be discussed, e.g., opening up design reviews to a wider community.

**Comment by dwsinger** [comment]
(I note we've been asked to move the affiliation change question to a different issue. Can we?)

**Comment by frivoal** [comment]
One more problem with the TAG appointment process in its current form is that it is ambiguous:

> The Team's choice of appointee(s) is subject to ratification by secret ballot by both the AB and the TAG, requiring a two-thirds approval from each.

How is that "two thirds" calculated? two thirds of eligible participants? Two thirds of those who do vote? Does silence mean approval or disapproval or neutrality?
